scanpy/neighbors/__init__.py

```diff
@@ -743,6 +888,10 @@ def compute_neighbors(
                 self._adata.shape[0],
                 self.n_neighbors,
             )
+        elif method == 'tsne':
+            self._distances, self._connectivities = _compute_connectivities_tsne(
+                knn_indices[:, 1:], knn_distances[:, 1:], n_jobs=-1
```

Comment:
Use `n_jobs=-1` meaning all available cores, since this is what the other functions do as well (I guess).